modules/tensor_mechanics/include/kernels/StressDivergenceTensors.h

```diff
@@ -75,7 +75,7 @@ class StressDivergenceTensors : public ALEKernel
   const unsigned int _temp_var;
 
   /// d(strain)/d(temperature), if computed by ComputeThermalExpansionEigenstrain
-  const MaterialProperty<RankTwoTensor> * const _deigenstrain_dT;
+  std::vector<const MaterialProperty<RankTwoTensor> *> _deigenstrain_dT;
```

Comment:
in existing parlance this would be `_deigenstrain_dargs`

Reply:
Or when we improve that, maybe `_deigenstrain_dcoupledvars`?